core/App/screens/ProofDetails.tsx

```diff
@@ -21,23 +22,39 @@ import { testIdWithKey } from '../utils/testable'
 
 type ProofDetailsProps = StackScreenProps<ProofRequestsStackParams, Screens.ProofDetails>
 
-const collapsedHeight = 120
+// const collapsedHeight = 120
+// const collapsedDuration = 150
 
 interface VerifiedProofProps {
   record: ProofExchangeRecord
   navigation: StackNavigationProp<ProofRequestsStackParams, Screens.ProofDetails>
-  isHistory?: boolean
 }
 
 interface UnverifiedProofProps {
   record: ProofExchangeRecord
 }
 
-const VerifiedProof: React.FC<VerifiedProofProps> = ({ record, navigation, isHistory }: VerifiedProofProps) => {
+// const gradientSVG = `<svg xmlns="http://www.w3.org/2000/svg" width="600px" height="200px" >
```

Comment:
Should it me moved to `CollapsibleView` component?

Reply:
done